### PR TITLE
Fix/bug a label level title appears in the gui at the opposite of camera and robot zappy gui

### PR DESCRIPTION
**Pull request by Thomaltarix**
I fixed the size of the text broadcast and level font.

Now, you need to look at the player's hitbox to see their level, It avoid to be spammed by player's level is you can see many players

### CODE REVIEW

**Review by Njord201 — DISMISSED** [review]

File: gui/src/Render/Render.cpp [hunk 1]
```diff
@@ -200,7 +200,7 @@ void Gui::Render::displayPlayers()
             else
                 DrawModelEx(team.getPlayerModel(), team.getPlayerPositionIn3DSpace(player.getId(), _gameData.get()->getMap()), ROTATION_AXIS_PLAYER, rotation, SCALE_PLAYER, team.getPlayerColor());
 
-            displayPlayerLevel(team, player);
+
```
Comment (Njord201) [inline gui/src/Render/Render.cpp:203]: Remove one line no ?

Comment (Thomaltarix) [reply gui/src/Render/Render.cpp:203]: In fact !

**Review by Njord201 — DISMISSED** [review]

File: gui/include/Render/Render.hpp [hunk 2]
```diff
@@ -249,10 +249,10 @@ class Gui::Render {
         /**
          * @brief Display player level.
          *
-         * @param team Team for the player 3d position.
          * @param player Player to display level.
+         * @param position Position to display the level.
          */
-        void displayPlayerLevel(Team &team, Player &player);
+        void displayPlayerLevel(Player &player, Vector3 position, Team &team);
```
Comment (mathieurobert1) [inline gui/include/Render/Render.hpp:255]: Update doxygen param

**Review by Njord201 — APPROVED** [review]

**Review by mathieurobert1 — CHANGES_REQUESTED** [review]
Just a simple update in documentation

File: gui/include/Render/Render.hpp (hunk 2 above)
Comment (Thomaltarix) [reply gui/include/Render/Render.hpp:256]: Done

**Review by Njord201 — APPROVED** [review]

**Review by mathieurobert1 — APPROVED** [review]
Nice work !

**Pull request merged**